rasa/core/training/interactive.py

```diff
@@ -168,19 +171,34 @@ class Abort(Exception):
                                       subpath=subpath)
     except ClientError:
         if is_new_action:
-            warning_questions = questionary.confirm(
-                "WARNING: You have created a new action: '{}', "
-                "which was not successfully executed. "
-                "If this action does not return any events, "
-                "you do not need to do anything. "
-                "If this is a custom action which returns events, "
-                "you are recommended to implement this action "
-                "in your action server and try again."
-                "".format(action_name))
-            await _ask_questions(warning_questions, sender_id, endpoint)
+            if action_name in NEW_TEMPLATES:
+                warning_questions = questionary.confirm(
+                    "WARNING: You have created a new action: '{0}', "
+                    "with matching template: {1}. "
+                    "This action will not return its message in this session, "
+                    "but the new utterance will be saved to your domain file "
+                    "when you exit and save this session. "
+                    "You do not need to do anything further. "
+                    "If you want to change the utterance or implement '{0}' as "
```

Comment:
i'm not sure this warning here makes sense? i think it's ok to just go on, the custom action stuff definitely doesn't really make sense

Reply:
I think it's good to tell the user that the `utter_{}` action and template were successfully created since it won't show up unless the session is saved, we could also do a debug or info logger statement for this instead maybe?

Reply:
no as in the first part makes sense, the part that comes after the line i commented on doesn't make sense imo

Reply:
yes, i removed that part now. I think the message was too long anyway